### PR TITLE
Desktop Skeleton Page

### PR DESCRIPTION
**Pull request by lancewf**
### :nut_and_bolt: Description: What code changed, and why?
We need a new page in automate to house the beautiful ui components and valuable data that make up the automate portion of the desktop offering. 

This PR is setting up the NGRX/store, routing, and API retrieval. Two crude UI components were added to show that the correct data is available. 

### :chains: Related Resources

https://github.com/chef/automate/issues/3107

### :athletic_shoe: How to Build and Test the Change
1. Start up the automate-ui locally with build component/automate-ui or the dev way with https://github.com/chef/automate/blob/master/dev-docs/ui-development.md. 
1. Go to https://a2-dev.test/dashboards/desktop

### :white_check_mark: Checklist

- [x] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [ ] Tests added/updated?
- [ ] Docs added/updated?
- [x] All commits have been signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### :camera: Screenshots, if applicable

Below are two crude desktop components that display the correct data to create the final UI components. 
image

The top component should end up looking something like below. 
image

The bottom component should end up looking something like below. 
image

### CODE REVIEW

File: components/automate-ui/src/app/modules/desktop/dashboard/dashboard.component.ts [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+    this.last24HourCheckInCount$ = this.checkInCountCollection$.pipe(
+      map(collection => last(collection.buckets))
+    );
+
+    this.unknownPercentage$ = this.last24HourCheckInCount$.pipe(
+        map(count => ((count.total - count.checkInCount) / count.total) * 100)
+      );
+
+    this.checkedInPercentage$ = this.last24HourCheckInCount$.pipe(
```
Comment (vjeffrey) [inline components/automate-ui/src/app/modules/desktop/dashboard/dashboard.component.ts]: if both the counts are 0, you'll end up with NaN here, bc 0/0 == NaN

Screen Shot 2020-03-26 at 16 36 06

Comment (lancewf) [reply components/automate-ui/src/app/modules/desktop/dashboard/dashboard.component.ts]: Oh, that should be %100. I will update it.

Comment (lancewf) [reply components/automate-ui/src/app/modules/desktop/dashboard/dashboard.component.ts]: I pushed a fix for that case.

Comment (SEAjamieD) [reply components/automate-ui/src/app/modules/desktop/dashboard/dashboard.component.ts]: I'm seeing these as %'s now 👍

**Pull request merged**